src/interfaces/hooks.ts

```diff
@@ -26,11 +26,11 @@ export interface Hooks {
     return: void;
   };
   preupdate: {
-    options: {channel: string};
+    options: {channel: string, version: string};
```

Comment:
Seems that channel and version are mutually exclusive, so should these be optional?

Reply:
Not necessarily because there's a version associated with the channel you're updating too